app/views/posts/show.html.erb

```diff
@@ -14,3 +14,8 @@
         <span> <%= comment.author.name %>: <%= comment.text%> </span>
     <% end %>
 </div>
+
+
+<div class="mx-auto" style="width: 200px;">
+<%=link_to "New Comment", new_user_post_comment_path(@user.id, @post.id), class:"btn btn-dark mb-4" %>
+</div>

```

Comment:
- [ ] Kindly consider the user should be able to add a like to posts by adding a button for like under the post.

Reply:
Done